test/runtests.jl

```diff
@@ -198,4 +215,94 @@ facts("Test integration of ODE with complex dependent variables") do
 
 end
 
+facts("Test jet transport: simple pendulum") do
+
+    function pendulum!(t, x, dx) #the simple pendulum ODE
+        dx[1] = x[2]
+        dx[2] = -sin(x[1])
+    end
+
+    const varorder = 2 #the order of the variational expansion
```

Comment:
I'm not sure if this matters or not, but it may be better to return `nothing` from the equations of motion for the multi-dimensional case, and to have the explicit return of `dx` for the one-dim case. If it doesn't matter, it may be a good idea to return `dx` always.